### ISSUE TITLE
Intellisense case-sensitive?

### ISSUE THREAD

**Issue by ahmetalpbalkan** [Microsoft/vscode-go#56]
I have a pkg called `rcode` contains exported constants `RcodeXxxxx` however if I type `rcode` it does not bring up autocomplete items.

![image](https://cloud.githubusercontent.com/assets/159209/11320277/ffcdc61a-9093-11e5-9112-396b50663a14.png)

Vs

![image](https://cloud.githubusercontent.com/assets/159209/11320279/0230dbd6-9094-11e5-8303-6b5109a66c79.png)

**Comment by newhook** [comment]
This is just how gocode works.

https://github.com/nsf/gocode/issues/91

**Comment by lukehoban** [comment]
@ahmetalpbalkan - It looks like you aren't actually getting completion lists from `vscode-go` at all here.  The icon with `abc` in it means you are seeing default results from VS code (just a list of words that appear anywhere in the text of the current file).

You should be seeing results with icons like these instead:

![screen shot 2015-11-21 at 4 02 00 pm](https://cloud.githubusercontent.com/assets/223467/11321191/3a0a0e86-9069-11e5-9a2b-b091d366ed1f.png)

@newhook Actually, as you can see in the screenshot above, `vscode-go` will show case0insensetive match results.

**Comment by lukehoban** [comment]
@ahmetalpbalkan I've closed this issue, but I'd still like to understand why you aren't seeing completion lists from `vscode-go` in your case.  Do you ever see results like in my screenshot, even in a simple standalone file?  What version of the extension do you have? 

```
grep 'version' ~/.vscode/extensions/lukehoban.Go/package.json
```

**Comment by lukehoban** [comment]
@ahmetalpbalkan BTW - the original issue you were seeing here, where you apparently weren't getting completion lists from `vscode-go`, was just fixed with #77, and that fix is in the `0.6.13` update.
